b2sdk/v2/transfer.py

```diff
@@ -0,0 +1,35 @@
+######################################################################
```

Comment:
the `set_thread_pool_size` methods are now part of the public API, but are going to be removed in v3. They are not tested, not even for existence and signature, which means CLI is probably gonna break when we release v3

Reply:
They're not part of the public API in the sense that their not mentioned in the docs as such. Is CLI gonna break when they're removed? I think it is not. I assume CLI is importing only from `v2`, and we're keeping this methods in `v2`, right?

Reply:
as it is now, the apiver methods use a deprecated private method defined in the "upstream" managers. If a developer simply removes it, these methods will stop working in v2, no test will break in the SDK and CLI will raise errors when using these options.

Reply:
It needs to be tested because when we release b2sdk v3, we might accidentally break something in the v2 interface and then b2cli installation which still uses b2sdk v2 (but a fresh v3-capable version of it) may break and since there are no tests for it at all, we won't know before the release.

Reply:
Alright, I'll add some unit test.

Reply:
Done in [b4c3758](https://github.com/Backblaze/b2-sdk-python/pull/297/commits/b4c3758f811af385a9a098ff281e03e6f67fc569)